src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1181 @@
+                -6.33e-11,
+                0.038331481,
+                78.17853009,
+                4.22e-09,
+            ],
+            16: [
+                -0.003702813,
+                -566.9002008,
+                0.003490278,
+                -10.24802298,
+                5.87e-05,
+                8.029940476,
+                0.000180849,
+                -1.67e-09,
+                -0.000223884,
+                -5.19e-06,
+                -0.699017368,
+                3.49e-06,
+                0.438254936,
+                0.010243428,
+                -2.60e-06,
+                312.1661769,
+                0.00564047,
+                1.39e-09,
+                0.098896296,
+                317.9402872,
+                1.27e-08,
+            ],
+        }
+
+        @self.Constraint(doc="sA surrogate equation")
+        def sA_cal(b):
+            return (
+                b.sA
+                == b.Xf * sA_coeffs[b.Nef.value][0]
+                + b.RR * sA_coeffs[b.Nef.value][1]
+                + b.Xf * b.RR * sA_coeffs[b.Nef.value][2]
+                + b.Tin * sA_coeffs[b.Nef.value][3]
+                + b.Tin * b.Xf * sA_coeffs[b.Nef.value][4]
+                + b.Tin * b.RR * sA_coeffs[b.Nef.value][5]
+                + b.Capacity * sA_coeffs[b.Nef.value][6]
+                + b.Capacity * b.Xf * sA_coeffs[b.Nef.value][7]
+                + b.Capacity * b.RR * sA_coeffs[b.Nef.value][8]
+                + b.Capacity * b.Tin * sA_coeffs[b.Nef.value][9]
+                + b.Pm * sA_coeffs[b.Nef.value][10]
+                + b.Pm * b.Xf * sA_coeffs[b.Nef.value][11]
+                + b.Pm * b.RR * sA_coeffs[b.Nef.value][12]
+                + b.Pm * b.Tin * sA_coeffs[b.Nef.value][13]
+                + b.Pm * b.Capacity * sA_coeffs[b.Nef.value][14]
+                + 1 * sA_coeffs[b.Nef.value][15]
+                + b.Pm**2 * sA_coeffs[b.Nef.value][16]
+                + b.Capacity**2 * sA_coeffs[b.Nef.value][17]
+                + b.Tin**2 * sA_coeffs[b.Nef.value][18]
+                + b.RR**2 * sA_coeffs[b.Nef.value][19]
+                + b.Xf**2 * sA_coeffs[b.Nef.value][20]
+            )
+
+        # Surrogate coefficients for calculating qs
+        qs_coeffs = {
+            8: [
+                -7.59e-06,
+                -34.3908768,
+                9.69e-06,
+                -0.131363139,
+                1.48e-07,
+                0.200569444,
+                0.002087075,
+                -1.31e-10,
+                -0.000541286,
+                -5.70e-06,
+                -0.109758232,
+                -3.05e-09,
+                0.000630081,
+                0.000228591,
+                -9.38e-08,
+                8.029964774,
+                0.002098453,
+                -2.59e-10,
+                0.000841111,
+                40.08888889,
+                -1.42e-12,
+            ],
+            10: [
+                1.59e-05,
+                -29.09136792,
+                -2.04e-05,
```

Comment:
I wonder if these values (and others shown) should be set up as variables and then fixed afterwards, leaving the model open for parameter estimation if ever needed (e.g., original developers of the model have new pilot data and want to re-evaluate parameter values).